website/docs/r/redis_instance.html.markdown

```diff
@@ -31,30 +31,19 @@ To get more information about Instance, see:
 
 ## Example Usage
 
-### Basic Usage
 ```hcl
-resource "google_redis_instance" "test" {
-  name           = "%s"
+resource "google_redis_instance" "cache" {
+  name           = "memory-cache"
   memory_size_gb = 1
 }
 ```
-
-### Full Usage
 ```hcl
-resource "google_compute_network" "test" {
-  name = "%s"
-}
-
-resource "google_redis_instance" "test" {
-  name           = "%s"
+resource "google_redis_instance" "cache" {
+  name           = "ha-memory-cache"
   tier           = "STANDARD_HA"
   memory_size_gb = 1
 
-  region                  = "us-central1"
-  location_id             = "us-central1-a"
-  alternative_location_id = "us-central1-f"
-  
-  authorized_network = "${google_compute_network.test.self_link}"
+  authorized_network = "${google_compute_network.auto-network.self_link}"
```

Comment:
what happened to location_id and alternative_location_id?

Reply:
Oh I totally just copied everything a test w/o region did for locations, which was include none of them. Added location stuff back.